_inc/client/components/jetpack-termination-dialog/dialog.jsx

```diff
@@ -109,8 +139,41 @@ class JetpackTerminationDialog extends Component {
 		);
 	}
 
+	renderSurvey() {
+		const { purpose } = this.props;
+		const { surveyAnswerId, surveyAnswerText } = this.state;
+		return (
+			<JetpackTerminationDialogSurvey
+				onSurveyAnswerChange={ this.handleSurveyAnswerChange }
+				purpose={ purpose }
+				surveyAnswerId={ surveyAnswerId }
+				surveyAnswerText={ surveyAnswerText }
+			/>
+		);
+	}
+
+	renderPrimaryButton() {
+		const { purpose } = this.props;
+		const { step, surveyAnswerId } = this.state;
+		return step === JetpackTerminationDialog.FEATURE_STEP ? (
+			<Button primary onClick={ this.handleContinueClick }>
+				{ __( 'Continue' ) }
+			</Button>
+		) : (
+			<Button
+				scary
+				primary
+				onClick={ this.handleTerminationClick }
+				disabled={ null === surveyAnswerId }
```

Comment:
remove this line if we do not want to force a survey answer

Reply:
Since the Calypso survey includes an option to skip, maybe we should too?
![image](https://user-images.githubusercontent.com/426388/76516886-89ce7800-645c-11ea-8134-7c094df2e77d.png)